backend/meeting/src/express-app.js

```diff
@@ -1,24 +1,21 @@
-const express = require("express");
-const cors = require("cors");
-const { meeting } = require("./api");
-const { CreateChannel } = require("./utils");
-
-module.exports = async app => {
-	app.use(express.json());
-	app.use(
-		cors({
-			origin: ["http://localhost:3000"],
-			methods: ["GET", "POST", "PUT", "DELETE"],
-			credentials: true,
-		})
-	);
-	app.use(express.static(__dirname + "/public"));
-
-	//api
-	// appEvents(app);
-
-	const channel = await CreateChannel();
-
-	meeting(app, channel);
-	// error handling
-};
+const express = require("express");
+const cors = require("cors");
+const { meeting } = require("./api");
+const { CreateChannel } = require("./utils");
+
+module.exports = async (app) => {
+	app.use(express.json());
+	app.use(
+		cors({
+			origin: ["http://localhost:3000"],

```

Comment:
This should be changed, no?

Reply:
It depends on the loadbalancer IP which changes evrytime we do terraform apply/destroy. But as I said I don't really think cors is a problem in production only in developement.